packages/cashscript/src/Transaction.ts

```diff
@@ -115,7 +115,13 @@ export class Transaction {
 
     this.builder.setLockTime(locktime);
 
-    const { inputs, outputs } = await this.getInputsAndOutputs(outs, options?.fee);
+    const { inputs, outputs } = await this.getInputsAndOutputs(
+      outs,
+      options?.fee,
+      DUST_LIMIT,
```

Comment:
It looks like I made a mistake in here. It should pass in `options?.minChange` here, but it looks like I forgot to do this in `0.3.2`. Could you replace `DUST_LIMIT` with `options?.minChange`?

Reply:
makes sense. happy to